sunpy/__init__.py

```diff
@@ -31,8 +32,6 @@ class UnsupportedPythonError(Exception):
     import builtins
     builtins._SUNPY_SETUP_ = False
```

Comment:
I wonder if we can remove this?

Reply:
Let me check.

Reply:
I've removed it, it seems to work locally.